src/Runner.Worker/JobExtension.cs

```diff
@@ -122,6 +123,18 @@ public async Task<List<IStep>> InitializeJob(IExecutionContext jobContext, Pipel
                         }
                     }
 
+                    var tokenPermissions = jobContext.Global.Variables.Get("system.github.token.permissions") ?? "";
+                    if (!string.IsNullOrEmpty(tokenPermissions))
+                    {
+                        context.Output($"##[group]GITHUB_TOKEN Permissions");
+                        var permissions = JsonConvert.DeserializeObject<Dictionary<string, string>>(tokenPermissions);
```

Comment:
consider catching exceptions here `JsonException` and logging to runner logs (Trace), may be good to not fail the job if we can't output the Token permissions. It likely won't happen, as we control the api side of this as well.

Reply:
Updated to catch all exceptions, display permissions is best effort.